v5_MIGRATION_GUIDE.md

```diff
@@ -149,41 +162,43 @@ class MyActivity : FragmentActivity() {
     
     // ALL OTHER ACTIVITY LAUNCH MODES 
     override fun onResume() {
-+       handleReturnToAppFromBrowser(requireActivity().intent)
++       handleReturnToAppFromBrowser(intent)
```

Comment:
Does this work in Kotlin? I thought you had to get the intent from the activity

Reply:
Oh maybe that's only in a fragment implementation nvm 🤦‍♀️

Reply:
The example class we currently have is an Activity:
```kotlin
class MyActivity : FragmentActivity() {
```

Would we want to make this a Fragment?

Reply:
Nah it doesn't need to be a fragment - it actually seems like with compose the Android ecosystem is now moving away from fragments